cp3pt0-deployment/setup_tenant.sh

```diff
@@ -306,10 +306,16 @@ function install_cs_operator() {
     msg "Installing IBM Foundational services operator into operator namespace - ${OPERATOR_NS}"
 
     info "checking if CommonService CRD exist in the cluster"
-    local is_CS_CRD_exist=$($OC get CustomResourceDefinition.apiextensions.k8s.io | (grep "commonservices.operator.ibm.com " || echo "fail"))
-    if [ "$is_CS_CRD_exist" != "fail" ]; then
+    local is_CS_CRD_exist=$(($OC get commonservice --ignore-not-found > /dev/null && echo exists) || echo fail)
+
+    if [ "$is_CS_CRD_exist" == "exists" ]; then
+        info "CommonService CRD exist"
         configure_cs_kind
+    elif [ "$is_CS_CRD_exist" != "fail" ]; then
+        error "Error getting CommonService CRD"
```

Comment:
Hi @qpdpQ 

I think `is_CS_CRD_exist` will only contain two results, either `exists` or `fail`, so we may not need this condition.